docs/testing.rst

```diff
@@ -41,8 +41,11 @@ a response or exception by shifting return values off of a queue.
     $client = new Client(['handler' => $handler]);
 
     // The first request is intercepted with the first response.
-    echo $client->request('GET', '/')->getStatusCode();
+    $response = $client->request('GET', '/');
+    echo $response->getStatusCode();
     //> 200
+    echo $response->getBody()->getContents();
```

Comment:
Using `getContents` is generally not a good idea, so I would like to see it gone from the documentation entirely.

Instead casting the body to string should be the recommended way.